prodtests/full-system-test/calib-workflow.sh

```diff
@@ -8,6 +8,7 @@ fi
 if [[ $BEAMTYPE != "cosmic" ]]; then
   has_detector_calib TPC && has_detectors TPC ITS TRD TOF && add_W o2-tpc-scdcalib-interpolation-workflow "$DISABLE_ROOT_OUTPUT --disable-root-input --pipeline $(get_N tpc-track-interpolation TPC REST)" "$ITSMFT_FILES"
   has_detector_calib ITS && has_detectors ITS && has_detectors_reco ITS && has_detector_matching PRIMVTX && [[ ! -z "$VERTEXING_SOURCES" ]] && add_W o2-calibration-mean-vertex-calibration-workflow
+  has_detector_calib TRD && has_detector ITS TPC TRD && add_W o2-calibration-trd-vdrift-exb
```

Comment:
Here it should be `has_detectors`, plural, no? Anyway, see my other comment.

Reply:
Yes, you are right. Thanks! I see you have already corrected that in https://github.com/AliceO2Group/O2DPG/pull/375

Reply:
Yes, I did, but of course the other PR is a complete change with respect to yours, and your calibration won't run for now in the FST. Is it ok?